File: QTD.SPA/src/app/_DtoModels/ILANercStandard_Link/ILANercStandard_LinkOptions.ts
export class ILANercStandard_LinkOptions {
  iLAId:any;
  stdId:any;
  nERCStdMemberId:any;
  creditHoursByStd:number;
  cehHours?:number;
  nercStdValues : NercStdValues[];


}

export class  NercStdValues {
  nERCStdMemberId:any;
  creditHoursByStd:number;
}

File: QTD2.Infrastructure/Model/ILA_NercStandard_Link/ILA_NercStandard_LinkOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.ILA_NercStandard_Link
{
    public class ILA_NercStandard_LinkOptions
    {
        public int ILAId { get; set; }

        public int StdId { get; set; }

        public int NERCStdMemberId { get; set; }

        public float CreditHoursByStd { get; set; }

        public List<NercStdValues> NercStdValues { get; set; }
    }

    public class NercStdValues
    {
        public int NERCStdMemberId { get; set; }

        public float CreditHoursByStd { get; set; }
    }
}
